.github/workflows/build.yml

```diff
@@ -0,0 +1,143 @@
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Get values for current commit
+      run: |
+        GIT_SHA_SHORT=$(echo ${{github.sha}} | cut -c 1-7)
+        GIT_REF=${{github.ref}}
+        GIT_BRANCH=${GIT_REF##*/}
+        echo "::set-env name=GIT_SHA_SHORT::$GIT_SHA_SHORT"
+        echo "::set-env name=GIT_BRANCH::$GIT_BRANCH"
+        echo "::set-env name=SHA_TAG::paas-$GIT_SHA_SHORT"
+        echo "::set-env name=BRANCH_TAG::paas-$GIT_BRANCH"
+
+    - name: Login to docker hub
+      run: echo "${{secrets.DOCKERHUB_PERSONAL_ACCESS_TOKEN}}" | docker login -u ${{secrets.DOCKERHUB_USERNAME}} --password-stdin
+
+    - name: "docker pull ${{env.DOCKER_IMAGE}}:${{env.BRANCH_TAG}}"
+      run: docker pull $DOCKER_IMAGE:BRANCH_TAG || true
+
+    - name: "docker pull ${{env.DOCKER_IMAGE}}-middleman:master"
+      run: docker pull $DOCKER_IMAGE-middleman:master || true
+
+    - name: "docker-compose build middleman"
+      run: $DOCKER_COMPOSE build middleman
+      env:
+        GIT_BRANCH: ${{env.BRANCH_TAG}}
+
+    - name: "docker-compose build"
+      run: $DOCKER_COMPOSE build
+      env:
+        GIT_BRANCH: ${{env.BRANCH_TAG}}
+
+    - name: Setup tests
+      run: |
+        $DOCKER_COMPOSE up --no-build -d
+        $DOCKER_COMPOSE exec -T web /bin/sh -c "./wait-for-command.sh -c 'nc -z db 5432' -s 0 -t 20"
+        $DOCKER_COMPOSE exec -T web /bin/sh -c "bundle exec rails db:setup"
+        $DOCKER_COMPOSE exec -T web /bin/sh -c "apk --no-cache add curl"
+        $DOCKER_COMPOSE exec -T web /bin/sh -c "bundle exec rake cc:setup"
+      env:
+        GIT_BRANCH: ${{env.BRANCH_TAG}}
+
+    - name: Run tests
+      run: |
+        $DOCKER_COMPOSE exec -T web /bin/sh -c 'bundle config --local disable_exec_load true'
+        $DOCKER_COMPOSE exec -T web /bin/sh -c 'bundle exec rake parallel:setup'
+        $DOCKER_COMPOSE exec -T web /bin/sh -c 'bundle exec rake "parallel:spec[,, -O .azure_parallel]"'
+      env:
+        GIT_BRANCH: ${{env.BRANCH_TAG}}
+
+    - name: Tag docker images
+      run: |
+        docker tag $DOCKER_IMAGE:$BRANCH_TAG $DOCKER_IMAGE:$SHA_TAG
+        docker tag $DOCKER_IMAGE_MAILER:$BRANCH_TAG $DOCKER_IMAGE_MAILER:$SHA_TAG
+        docker tag $DOCKER_IMAGE_GEOCODE:$BRANCH_TAG $DOCKER_IMAGE_GEOCODE:$SHA_TAG
+        docker tag $DOCKER_IMAGE_JOBS:$BRANCH_TAG $DOCKER_IMAGE_JOBS:$SHA_TAG
+
+    - name: Push docker images to Docker Hub
+      run: |
+        docker push $DOCKER_IMAGE:$SHA_TAG
+        docker push $DOCKER_IMAGE:$BRANCH_TAG
+        [[ "$BRANCH_TAG" == "master" ]] && docker push $DOCKER_IMAGE-middleman:$BRANCH_TAG
+        docker push $DOCKER_IMAGE_MAILER:$SHA_TAG
+        docker push $DOCKER_IMAGE_MAILER:$BRANCH_TAG
+        docker push $DOCKER_IMAGE_GEOCODE:$SHA_TAG
+        docker push $DOCKER_IMAGE_GEOCODE:$BRANCH_TAG
+        docker push $DOCKER_IMAGE_JOBS:$SHA_TAG
+        docker push $DOCKER_IMAGE_JOBS:$BRANCH_TAG
```

Comment:
As discussed over Zoom, we'll remove these pushes and only push up the main image (and middleman), and then configure the `bg-jobs` background task to run off of the main image and specifying the command to run (`sidekiq`) on all the queues.

Reply:
This is now ready to be merge to master... could this be reviewed please....?